public/conferences/paris-2024/mill-builds-scala3.md

```diff
@@ -16,7 +16,7 @@ Mill is a popular build tool used in Scala projects, where build definitions are
 ### Jamie Thompson
 
 - photoRelPath: /images/profiles/paris-2024/jThompson.webp
-- job: Scala 3 compiler contributor
+- job: Contributor @ Scala 3 compiler
```

Comment:
I'm not really sure we can have the `job @ company` pattern here

Reply:
This is a quickfix to this:
<img width="501" alt="Capture d’écran 2024-09-12 à 18 01 45" src="https://github.com/user-attachments/assets/c56e67f4-585f-42a9-a7f2-094e4c3b8fdb">
but maybe this is not a good option.

Reply:
Arf, okay let's go with it, I'll see on how to improve this